.travis.yml

```diff
@@ -111,45 +112,54 @@ matrix:
       - zip -qry onscripter-ru_linux64_r${BUILD}.zip onscripter-ru || exit 1
       - rm onscripter-ru ; cd ..
 
-  - os: linux
+  - os: windows
```

Comment:
I am fine with fixing hosted build on Windows, but CI must build on Linux, as this is our primary test platform for cross builds.